src/main/java/frc/robot/subsystems/Dashboard.java

```diff
@@ -0,0 +1,71 @@
+        var frAngle = swerveAnglesLayout.add("FR", 0).withPosition(2, 0).withSize(2,2).withWidget(BuiltInWidgets.kTextView);
+        var flAngle = swerveAnglesLayout.add("FL", 0).withPosition(0, 0).withSize(2,2).withWidget(BuiltInWidgets.kTextView);
+        var brAngle = swerveAnglesLayout.add("BR", 0).withPosition(2, 2).withSize(2,2).withWidget(BuiltInWidgets.kTextView);
+        var blAngle = swerveAnglesLayout.add("BL", 0).withPosition(0, 2).withSize(2,2).withWidget(BuiltInWidgets.kTextView);
+        var odometer = driveBaseLayout.add("Odometer", "(0,0)").withPosition(0, 2).withSize(4,1).withProperties(null).withWidget(BuiltInWidgets.kTextView);
+        var armLayout = tab.getLayout("Arm", BuiltInLayouts.kGrid).withPosition(9, 2).withSize(8,1);
+        var shoulderAngle = armLayout.add("ShoulderAngle", 0).withPosition(0, 0).withSize(4,1).withWidget(BuiltInWidgets.kTextView);
+        var extensionDistance = armLayout.add("ExtensionDistance", 0).withPosition(4, 0).withSize(4, 1).withWidget(BuiltInWidgets.kTextView);
+        var intakeLayout = tab.getLayout("IntakeLayout", BuiltInLayouts.kGrid).withPosition(9, 5).withSize(8, 4);
+        var wristAngle = armLayout.add("WristAngle", 0).withPosition(0, 0).withSize(4, 1).withWidget(BuiltInWidgets.kTextView);
+        var twistAngle = armLayout.add("TwistAngle", 0).withPosition(0, 2).withSize(4, 1).withWidget(BuiltInWidgets.kTextView);
+        var hasGamePiece = armLayout.add("HasGamePiece", false).withPosition(4, 0).withSize(4, 1).withWidget(BuiltInWidgets.kBooleanBox);
+        
+        // What is a default value for a camerastream?
+        //var camera = tab.add("Camera", false).withPosition(0, 0).withSize(10, 10).withWidget(BuiltInWidgets.kCameraStream);
+
+        var settingsTab                 = Shuffleboard.getTab("Settings");
+
+        //Drive
+        var driveSettingsLayout         = settingsTab.getLayout("Drive Subsystem", BuiltInLayouts.kGrid).withPosition(0, 0).withSize(4, 2);
+        var setFLAngle                  = driveSettingsLayout.add("FL Angle", 0.0).withPosition(0, 0).withSize(2,2).withWidget(BuiltInWidgets.kTextView);
+        var setFRAngle                  = driveSettingsLayout.add("FR Angle", 0.0).withPosition(2, 0).withSize(2,2).withWidget(BuiltInWidgets.kTextView);
+        var setBLAngle                  = driveSettingsLayout.add("BL Angle", 0.0).withPosition(0, 2).withSize(2,2).withWidget(BuiltInWidgets.kTextView);
+        var setBRAngle                  = driveSettingsLayout.add("BR Angle", 0.0).withPosition(2, 2).withSize(2,2).withWidget(BuiltInWidgets.kTextView);
+        
+        //Vision
+        
+        //Arm
+        var armSettingsLayout           = settingsTab.getLayout("Arm Subsystem", BuiltInLayouts.kGrid).withPosition(0, 4).withSize(4, 2);
+        var setArmExtension             = armSettingsLayout.add("Arm Extension", 0.0);
+        var setArmAngle                 = armSettingsLayout.add("Arm Angle", 0.0);
```

Comment:
We need both a minimum and maximum arm angle to define the maximum travel range of the arm.